iceberg/iceberg-handler/src/test/results/positive/vectorized_iceberg_merge_mixed.q.out

```diff
@@ -957,7 +957,7 @@ Stage-6
                                 Output:["_col0","_col1","_col2","_col3","_col4","_col5","_col6","_col7","_col8","_col9","_col10","_col11","_col12","_col13","_col14","_col15","_col16","_col17","_col18","_col19","_col20","_col21","_col22","_col23","_col24","_col25","_col26"]
                                 Filter Operator [FIL_46] (rows=2 width=700)
                                   predicate:((ss_sold_date_sk = 2451181) and ss_item_sk is not null and ss_customer_sk is not null)
-                                  TableScan [TS_2] (rows=2 width=58###)
+                                  TableScan [TS_2] (rows=2 width=55795)
```

Comment:
looks like masking doesn't work here
````
-- Mask width
--! qt:replace:/(width=58)\d+/$1###/
````

Reply:
Fixed in https://github.com/apache/hive/pull/4824/commits/00f4e99be7d6a90e4f8d01f4565225f6088a1c5f